imagersite/imager_images/views.py

```diff
@@ -1,3 +1,86 @@
 from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.http import HttpResponse
+from .models import Photo, Album
+from django.views.generic.edit import ModelFormMixin, ProcessFormView
+from django.views.generic.detail import SingleObjectTemplateResponseMixin
+import datetime
 
-# Create your views here.
+LIBRARY_TEMPLATE = 'library.html'
+
+
+@login_required(login_url='/login')
+def Library(request, *args, **kwargs):
```

Comment:
If you aren't expecting any args or kwargs (because you didn't put any into your url for this view) then you don't need to add those to the call signature for this view function.  Only the request is required.